### PR TITLE
ibus-rime: update to 1.5.0

### PR DESCRIPTION
**Pull request by svenper**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/librime/template [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-version=1.5.3
-revision=4
+version=1.7.2
+revision=1
 build_style=cmake
 configure_args="-DCMAKE_BUILD_TYPE=None -DENABLE_LOGGING=OFF -DBUILD_TEST=ON"
 makedepends="boost-devel leveldb-devel marisa-devel opencc-devel
- yaml-cpp-devel gtest-devel"
+ yaml-cpp-devel gtest-devel capnproto-devel"
+hostmakedepends="pkg-config"
```
Comment (ericonr) [inline srcpkgs/librime/template:9]: You probably need `capnproto` in `hostmakedepends` as well. And they should go above `makedepends`

Comment (svenper) [reply srcpkgs/librime/template]: Hmm, ARM still fails with those changes?

**Comment by ericonr** [comment]
They are using the target's capnproto binary :(

`/usr/aarch64-linux-gnu/usr/bin/capnp`

If it doesn't actually need `capnproto-devel` in `makedepends`, that might solve the issue handily, I think.

**Comment by svenper** [comment]
It does unfortunately require -devel too.

```text
=> librime-1.7.2_1: running post-install hook: 06-strip-and-debug-pkgs ...
/usr/bin/aarch64-linux-musl-strip: Unable to recognise the format of the input file `/destdir/aarch64-linux-musl/librime-1.7.2/usr/lib/librime.so.1.7.2'
=> ERROR: librime-1.7.2_1: post-install_06-strip-and-debug-pkgs: '$STRIPCMD --strip-unneeded "$f"' exited with 1
```

**Comment by sgn** [comment]
This diff makes package cross-compilable, but I didn't test the result (64bit cross to 32 bit)
```diff
diff --git a/srcpkgs/librime/template b/srcpkgs/librime/template
index 4f1a4b25dd..08ad94ca21 100644
--- a/srcpkgs/librime/template
+++ b/srcpkgs/librime/template
@@ -3,10 +3,11 @@ pkgname=librime
 version=1.7.2
 revision=1
 build_style=cmake
-configure_args="-DCMAKE_BUILD_TYPE=None -DENABLE_LOGGING=OFF -DBUILD_TEST=ON"
+configure_args="-DCMAKE_BUILD_TYPE=None -DENABLE_LOGGING=OFF -DBUILD_TEST=ON
+ -DCAPNP_EXECUTABLE=/usr/bin/capnp -DCAPNPC_CXX_EXECUTABLE=/usr/bin/capnpc-c++"
 hostmakedepends="pkg-config capnproto"
 makedepends="boost-devel leveldb-devel marisa-devel opencc-devel
- yaml-cpp-devel gtest-devel capnproto-devel"
+ yaml-cpp-devel gtest-devel capnproto-devel xorgproto"
 short_desc="Rime Input Method Engine"
 maintainer="Yuxuan Shui <yshuiv7@gmail.com>"
 license="BSD-3-Clause"
```

File: srcpkgs/librime/template [hunk 2]
```diff
@@ -1,17 +1,19 @@
 # Template file for 'librime'
 pkgname=librime
-version=1.5.3
-revision=4
+version=1.7.2
```
Comment (ericonr) [inline srcpkgs/librime/template:3]: 1.7.3 is out

Comment (svenper) [reply srcpkgs/librime/template:3]: Is that not a pre-release version?  
https://github.com/rime/librime/releases/tag/1.7.3

Comment (ericonr) [reply srcpkgs/librime/template:3]: It says pre-release, but it also includes only two commits fixing undefined behavior. So I'd prefer that we move to it. Might make sense to ask upstream why they tagged it that way.

Comment (sgn) [reply srcpkgs/librime/template]: The second one (buffer overflow) was in a plugin that we don't have. (do we in anyway pull that plugin in?)
The first one is near-null-pointer dereference, I think.

**Pull request merged**